CMakeLists.txt

```
@@ -17,7 +17,7 @@ SET(CMAKE_CXX_STANDARD 17)
 
 SET(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
 if (NOT MSVC)
-  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -fdiagnostics-color=always")
+  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -fdiagnostics-color=always -fno-lto")  ##-fno-lto to fix 'bytecode stream version'
```

```suggestion
  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -fdiagnostics-color=always")
```
There is 
```
if(CMAKE_BUILD_TYPE MATCHES "Release")
  include(CheckIPOSupported)
```
block below. Maybe comment it out instead? It's not recommended to use raw compiler flags in CMake.

---

accept, fixed